tuning-crc-openshift-cluster/apply-kernel-parameters.sh

```diff
@@ -0,0 +1,5 @@
+SSH_KEYS_OF_MASTER_NODE=../id_rsa_crc
+set -x
```

Comment:
All the scripts should have `set -exuo pipefail` at top so we can easily exit as soon as possible when there is issue with a specific script.

Reply:
I've done that in https://github.com/cfergeau/snc/commit/1370fe5ad6542e4497779b7208f65fec6d7e03b8